File: 0268-missing-number/0268-missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    let n=nums.length
    let sum=0
    for(i=0;i<n;i++){
        sum+=nums[i]
    }
    return (n*(n+1)/2)-sum
};

File: 0268-missing-number/0268-missing-number.py
class Solution(object):
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        sum=(len(nums)*(len(nums)+1))/2
        print(sum)
        for i in range(len(nums)):
            sum-=nums[i]
        return sum